Helper/AlgoliaHelper.php

```diff
@@ -152,7 +159,7 @@ public function getIndex(string $name)
     }
 
     /**
-     * @return mixed
+     * @return ListIndicesResponse
```

Comment:
I like that we're adding more specificity here but I've wondered about the response objects in the PHP client API. 

I see the API call for `\Algolia\AlgoliaSearch\Api\SearchClient::listIndices` can return:
```
\Algolia\AlgoliaSearch\Model\Search\ListIndicesResponse|array<string, mixed>
```

Do we need to provide a union type in our code as well? When would one be returned vs. the other?

I would hate for an unexpected type mismatch to create an issue here.

Reply:
You're right with this, indeed, the php client most probably return an array, changing this right now